test/puppetlabs/puppetdb/integration/reports/report_storage.clj

```diff
@@ -0,0 +1,75 @@
+(ns puppetlabs.puppetdb.integration.reports.report-storage
+  (:require [clojure.test :refer :all]
+            [me.raynes.fs :as fs]
+            [puppetlabs.puppetdb.integration.fixtures :as int]
+            [puppetlabs.puppetdb.testutils.services :as svc-utils]
+            [puppetlabs.puppetdb.cheshire :as json]))
+
+(defn get-href [pdb suffix]
+  (-> pdb
+      int/root-url-str
+      (str suffix)
+      svc-utils/get-ssl
+      :body))
+
+(deftest ^:integration metrics-and-logs-storage
+  (with-open [pg (int/setup-postgres)
+              pdb (int/run-puppetdb pg {})
+              ps (int/run-puppet-server-as "something_silly" [pdb] {})]
+    (testing "Initial agent run, to populate puppetdb with data to query"
+      (int/run-puppet-as "my_agent" ps pdb
+                         (str "notify { \"hi\":"
+                              "  message => \"Hi my_agent\" "
+                              "}")
+                         ["--noop"])
+      (let [result (int/entity-query pdb "/reports"
+                                     ["=" "certname" "my_agent"]
+                                     {"order_by" (json/generate-string [{"field" "receive_time"
+                                                                         "order" "desc"}])})
+            [event :as events] (remove #(= (:resource_type %) "Schedule")
```

Comment:
Can we replicate the 'this is weird' comment from the ruby test? 'cause it's weird.